src/app/search/FilterHelp.tsx

```diff
@@ -0,0 +1,700 @@
+            {destinyVersion === 1 && (
+              <tr>
+                <td>
+                  <span>is:haslight</span>
+                </td>
+                <td>{t('Filter.ContributeLight')}</td>
+              </tr>
+            )}
+
+            {destinyVersion === 2 && (
+              <tr>
+                <td>
+                  <span>is:haspower</span>
+                </td>
+                <td>{t('Filter.ContributePower')}</td>
+              </tr>
+            )}
+            <tr>
+              <td>
+                <span>is:weapon</span> <span>is:armor</span> <span>is:ghost</span>
+                <span>is:vehicle</span> {destinyVersion === 2 && <span>is:ships</span>}
+                {destinyVersion === 2 && <span>is:engrams</span>}
+                {destinyVersion === 2 && <span>is:consumables</span>}
+                {destinyVersion === 2 && <span>is:modifications</span>}
+                {destinyVersion === 2 && <span>is:shaders</span>}
+                {destinyVersion === 1 && <span>is:cosmetic</span>}
+              </td>
+              <td>{t('Filter.Categories')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:equipment</span> <span>is:equippable</span>
+              </td>
+              <td>{t('Filter.Equipment')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:transferable</span> <span>is:movable</span>
+              </td>
+              <td>{t('Filter.Transferable')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:equipped</span>
+              </td>
+              <td>{t('Filter.Equipped')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:weapon</span> <span>is:class</span>
+                {destinyVersion === 1 && <span>is:primary</span>}
+                {destinyVersion === 1 && <span>is:special</span>}
+                {destinyVersion === 1 && <span>is:heavy</span>}
+                {destinyVersion === 2 && <span>is:kinetic</span>}
+                {destinyVersion === 2 && <span>is:energy</span>}
+                {destinyVersion === 2 && <span>is:power</span>}
+              </td>
+              <td>{t('Filter.WeaponClass')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:armor</span> <span>is:helmet</span> <span>is:gauntlets</span>
+                <span>is:chest</span> <span>is:leg</span> <span>is:classitem</span>
+              </td>
+              <td>{t('Filter.ArmorCategory')}</td>
+            </tr>
+            <tr>
+              <td>
+                <span>is:common</span> <span>is:uncommon</span> <span>is:rare</span>
+                <span>is:legendary</span> {destinyVersion === 1 && <span>is:sublime</span>}
+                <span>is:exotic</span> <span>is:white</span> <span>is:green</span>{' '}
+                <span>is:blue</span>
+                <span>is:purple</span> <span>is:yellow</span>
```

Comment:
`                <span>is:blue</span> <span>is:purple</span> <span>is:yellow</span>`